ddg_api.py

```diff
@@ -7,27 +7,48 @@
 cors = CORS(app, resources={r"/*":
                             {"origins": "http://localhost:3000"}})
 
+global saved_mols
+# Can't serialize sets using json
+saved_mols = []
+
 
 @app.route("/")
 def hello_world():
     return "<p>Hello, World!</p>"
 
 
+# Pass molecule IDs as queries: /save?r1=A01&r2=B10
+@app.route("/save", methods=['POST'])
```

Comment:
Should you define what this function (and the next one) does with """...."""?

Reply:
yes